Sources/Deboogger.swift

```diff
@@ -52,14 +52,9 @@ public final class Deboogger {
 
     private var gesture: DebooggerGesture = .init()
 
-    private lazy var assistiveButtonWindow: UIWindow = {
-        let size = AssistiveButton.Layout.size
-        let window = UIWindow(frame: CGRect(x: 0, y: 0, width: size, height: size))
-        window.windowLevel = AssistiveButtonWindowLevel
-        window.rootViewController = self.assistiveButtonPresenterViewController
-        window.isHidden = true
-        return window
-    }()
+    private var keyWindow: UIWindow {
+        UIApplication.shared.keyWindow ?? UIWindow()
```

Comment:
I would prefer `Deboogger` to be initialized with a target window instead. 
There're cases in which `keyWindow` will be `nil`, despite a main window already existing.
Also, somebody may try to setup Deboogger before creating a window, which of course won't & shouldn't work, but this error won't be communicated in any meaningful way.